firstvoices/backend/tests/test_apis/test_site_features_api.py

```diff
@@ -164,3 +164,40 @@ def test_duplicate_key_validation(self):
         )
 
         assert response.status_code == 400
+
+    @pytest.mark.django_db
+    def test_site_feature_keys_only_unique_per_site(self):
```

Comment:
Please also add a test case checking that adding non-unique keys for a single site end up in a validation error.

Reply:
The test above this one already checks for duplicate keys

Reply:
Oh sorry, yeah the naming confused me for this test. Approving this now.